Reyes-JohnRuiz/script.js

```diff
@@ -0,0 +1,54 @@
+let count = 0;
+let intervalId;
+let isTimerStarting = false;
+
+let counterElement = document.getElementById("counter");
+let startButton = document.getElementById("start");
+let pauseButton = document.getElementById("pause");
+let resumeButton = document.getElementById("resume");
+let stopButton = document.getElementById("stop");
+
+function startCounter() {
```

Comment:
I suggest you use guard clause approach here

See reference:
https://learningactors.com/javascript-guard-clauses-how-you-can-refactor-conditional-logic/

It could be like this below:
```
function startCounter() {
  if (isTimerStarting) return;

  intervalId = setInterval(incrementCounter, 1000);
  startButton.disabled = true;
  pauseButton.disabled = false;
  resumeButton.disabled = true;
  stopButton.disabled = false;
}
```